src/src.pro

```diff
@@ -2,6 +2,8 @@
 TEMPLATE = app
 CONFIG += qt thread silent
 
+DEFINES += QBT_APP_NAME=\\\"qBittorrent\\\"
```

Comment:
Please add to the top-level CMakeListst.txt after the `project()` call (probably near the version-related stuff):
```cmake
add_definitions(-DQBT_APP_NAME="${CMAKE_PROJECT_NAME}")
```

Reply:
Ok!

Reply:
Thanks.